src/Emulator/Host/fs.h

```diff
@@ -0,0 +1,108 @@
+#pragma once
+#include <types.h>
+
+#include <cstddef>
+#include <cstdint>
+#include <cstdio>
+#include <filesystem>
+#include <optional>
+#include <vector>
+
+#include "Lib/Threads.h"
+
+namespace Emulator::Host::Fs {
+
+class MntPoints {
+  public:
+    struct MntPair {
+        std::string host_path;
+        std::string guest_path;  // e.g /app0/
+    };
+
+    MntPoints() {}
+    virtual ~MntPoints() {}
+    void mount(const std::string& host_folder, const std::string& guest_folder);
+    void unMount(const std::string& path);
```

Comment:
unmount is one word, naming convention dictates to use `unmount` and `unmountAll`

Reply:
ok